### PR TITLE
ROX-20148 - Remove operator installation from development scripts

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description

:fire: :fire: :fire: 

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources

## Test manual

**TODO:** Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```

### CODE REVIEW

File: dev/env/manifests/shared/00-namespace.yaml [hunk 1]
```diff
@@ -2,3 +2,8 @@ kind: Namespace
 apiVersion: v1
 metadata:
   name: "$ACSCS_NAMESPACE"
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: $STACKROX_OPERATOR_NAMESPACE
```
Comment (kovayur) [inline dev/env/manifests/shared/00-namespace.yaml:9]: Can we use `ACSCS_NAMESPACE` only and rename it from `acscs` to `rhacs` like for the real clusters?

File: dev/env/manifests/rhacs-operator/quay/01-catalogsource.yaml [hunk 2]
```diff
@@ -1,12 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
-kind: CatalogSource
-metadata:
-  name: stackrox-operator-test-index
-  namespace: "$STACKROX_OPERATOR_NAMESPACE"
-spec:
-  sourceType: grpc
-  image: "${STACKROX_OPERATOR_INDEX_IMAGE}"
```
Comment (kovayur) [inline dev/env/manifests/rhacs-operator/quay/01-catalogsource.yaml:8]: Remove this variable from lib.sh

File: dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml [hunk 3]
```diff
@@ -1,13 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: stackrox-operator-test-subscription
-  namespace: "$STACKROX_OPERATOR_NAMESPACE"
-spec:
-  channel: $STACKROX_OPERATOR_CHANNEL
```
Comment (kovayur) [inline dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml:7]: I see another usages that should be removed as well:
1. 11-operator-channel.env
1. lib.sh

File: dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml [hunk 4]
```diff
@@ -1,13 +0,0 @@
...
-  name: rhacs-operator
-  installPlanApproval: Automatic
-  source: redhat-operators
-  sourceNamespace: openshift-marketplace
-  config:
-    resources: $RHACS_OPERATOR_RESOURCES
```
Comment (kovayur) [inline dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml:13]: Please cleanup lib.sh and the occurrences of `RHACS_OPERATOR_RESOURCES_DEFAULTS`

**Review by kovayur — APPROVED** [review]
Looks good already, but I believe we could clean up even more

File: dev/env/manifests/shared/00-namespace.yaml (hunk 1 above)
Comment (SimonBaeumer) [reply dev/env/manifests/shared/00-namespace.yaml:9]: Yes, but outside of this PR. Here I am only removing the development scripts, the proposed change needs more testing.

File: dev/env/manifests/rhacs-operator/quay/01-catalogsource.yaml (hunk 2 above)
Comment (SimonBaeumer) [reply dev/env/manifests/rhacs-operator/quay/01-catalogsource.yaml:8]: done

File: dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml (hunk 3 above)
Comment (SimonBaeumer) [reply dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml:7]: done

File: dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml (hunk 4 above)
Comment (SimonBaeumer) [reply dev/env/manifests/rhacs-operator/marketplace/03-subscription.yaml:13]: nice catch thx, done

**Pull request merged**